packages/grid/src/Grid.tsx

```diff
@@ -226,6 +228,10 @@ export const Grid = function Grid<T>(props: GridProps<T>) {
     headerIsFocusable ? "header" : "body"
   );
 
+  const [sortByColumnId, setSortByColumnId] =
+    useState<GridColumnProps["id"]>("");
```

Comment:
Could "" be a valid id?

Reply:
Hmm not sure how that could be done, as on first sortable column header click, that's when the `colHeaderId` to be sorted gets set, it would be difficult to have an initial `colHeaderId` set as the initial state value - do you have something else in mind?